src/store/store.js

```diff
@@ -0,0 +1,96 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import axios from 'axios'
+
+Vue.use(Vuex)
+
+const store = new Vuex.Store({
+  state: {
+    personas: [],
+    persona: {
+      _id: '',
+      firstName: '',
+      lastName: '',
+      gender: '',
+      born: '',
+      age: '',
+      noIdent: '',
+      children: '',
+      position: '',
+      salary: '',
+      race: ''
+    }
+  },
+  actions: {
+    listingPerson: function ({commit}) {
+      axios.get('/persona').then((response) => {
+        commit('LISTING_PERSON', {persona: response.data})
```

Comment:
Es una buena practica usas constantes para los nombres de las mutations. https://vuex.vuejs.org/guide/mutations.html#using-constants-for-mutation-types